torchtext/functional.py

```diff
@@ -10,10 +10,10 @@
 ]
 
 
-def to_tensor(input: Union[List[int], List[List[int]]], padding_value: Optional[int] = None, dtype: Optional[torch.dtype] = torch.long) -> Tensor:
+def to_tensor(input: Any, padding_value: Optional[int] = None, dtype: Optional[torch.dtype] = torch.long) -> Tensor:
```

Comment:
IIRC, this function is going to be public API in the next release. I am aware adding doctoring is in todo list of the PR, but I think it's better done in the same PR, rather than late PR.